packages/graphback-codegen-schema/src/targetSchemaContext.ts

```diff
@@ -144,24 +160,12 @@ export const buildTargetContext = (input: InputModelTypeContext[]) => {
             "type": 'Type',
             "relation": `${f.name}: ${f.type}`
           })
-          relations.push({
```

Comment:
[Requires Discussion] So our approach is to not generate bidirectional relationships, but we do not provide validation on input schema.

Reply:
In what scenario would you consider the input schema invalid? I'm not sure I understand. The following different datamodels will produce the same database structure.

```gql
type Note {
  id: ID!
  title: String!
  description: String
  comments: [Comment]!
}

type Comment {
  id: ID!
  description: String
  note: Note!
}
```

```gql
type Note {
  id: ID!
  title: String!
  description: String
}

type Comment {
  id: ID!
  description: String
  note: Note!
}
```